client/js/power-calibration.js

```diff
@@ -1,21 +1,22 @@
+/* global io */
 const socket = io();
-
+// eslint-disable-next-line no-unused-vars
```

Comment:
If we want, we can relax the linters constraints but let's try not to disable linter for each line